ui/page/components/coinformat.go

```diff
@@ -115,31 +115,27 @@ func getIndexUnit(amount string) int {
 // LayoutBalance aligns the main and sub DCR balances horizontally, putting the sub
 // balance at the baseline of the row.
 func LayoutBalance(gtx layout.Context, l *load.Load, amount string) layout.Dimensions {
-	return formatBalance(gtx, l, amount, values.TextSize20, defaultScale, l.Theme.Color.Text, false, false)
+	return formatBalance(gtx, l, amount, values.TextSize20, l.Theme.Color.Text, false, false)
 }
 
 func LayoutBalanceWithUnit(gtx layout.Context, l *load.Load, amount string) layout.Dimensions {
-	return formatBalance(gtx, l, amount, values.TextSize20, defaultScale, l.Theme.Color.PageNavText, false, true)
+	return formatBalance(gtx, l, amount, values.TextSize20, l.Theme.Color.PageNavText, false, true)
 }
 
 func LayoutBalanceWithUnitSize(gtx layout.Context, l *load.Load, amount string, mainTextSize unit.Sp) layout.Dimensions {
-	return formatBalance(gtx, l, amount, mainTextSize, defaultScale, l.Theme.Color.PageNavText, false, true)
+	return formatBalance(gtx, l, amount, mainTextSize, l.Theme.Color.PageNavText, false, true)
 }
 
 func LayoutBalanceWithUnitSizeBoldText(gtx layout.Context, l *load.Load, amount string, mainTextSize unit.Sp) layout.Dimensions {
-	return formatBalance(gtx, l, amount, mainTextSize, .85, l.Theme.Color.PageNavText, true, true)
+	return formatBalance(gtx, l, amount, mainTextSize, l.Theme.Color.PageNavText, true, true)
```

Comment:
The default scale is .7, but this guy was using .85. I see that this `LayoutBalanceWithUnitSizeBoldText` was added by @Sirmorrison in #247. Let's get his thoughts on this change to use .7 instead of .85.

Reply:
Its used by the mixer layout to scale the size of the main text and subtext to the update UI.
<img width="449" alt="image" src="https://github.com/crypto-power/cryptopower/assets/27733432/cb39b69c-9623-4891-afdd-5bab1e2710b0">

Reply:
I think it's fine to use the same scale all through. However, I'm not sure the default scale should be .7. From what I see in many places on figma, the significant part of the amount text uses 18px while the insignificant part uses 14px, which is a scale of 0.78 (14/18). Please verify and adjust accordingly @JustinBeBoy.